lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,34 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
+ * @function
+ * @public
+ *
+ * @description
+ * This function returns undefined if there was no last flash.
+ *
+ * @returns {String} the last flash uuid
+ *
+ * @example
+ * const uuid = flashState.getLastFlashUuid();
+ */
+exports.getFlashUuid = () => {
+  return Store.getState().toJS().flashUuid;
+};
+
+/**
+ * @summary Generate flash uuid
+ * @function
+ * @public
+ *
+ * @example
+ * flashState.generateFlashUuid();
+ */
+exports.generateFlashUuid = () => {
```

Comment:
I don't think we should expose this as something the user can call, since it means it can easily get out of sync. I think the UUID should be generated when the `SET_FLASHING_FLAG` is called. What do you think?